mkdocs.yml

```
@@ -0,0 +1,6 @@
+site_name: Playbook
+site_description: Playbook Documentation
+nav:
+  - "Home": "README.md"
+  - "Upgrade Guide": "upgrade-guide.md"
+  - "Change Log": "CHANGELOG.md"
```

```suggestion
  - "Change Log": "CHANGELOG.md"
plugins:
  - techdocs-core
```

Found out this morning or so that we might need to add in this line to the mkdocs.yml. Main thing is for styling. Here's two images without and with the line.

https://nitro.powerhrg.com/connect/api/v4/files/2105558/body

https://nitro.powerhrg.com/connect/api/v4/files/2105563/body

I can go back and add it for any doc we already merged. Thoughts?

@ryanmrodriguez

---

Sounds good